.gitignore

```diff
@@ -12,5 +12,6 @@ Publish
 
 /packages
 /TestResults
+App_Data/
```

Comment:
What is this needed for? I'd like to keep these sample files checked in. They're already excluded from the zipped site extension.